MUXSDKStats/Podfile

```diff
@@ -6,13 +6,15 @@ target 'MUXSDKStats' do
   # use_frameworks!
 
   # Pods for MUXSDKStats
-  pod 'Mux-Stats-Core', '~>3.0.3'
+  #pod 'Mux-Stats-Core', '~>3.0.3'
+  pod 'Mux-Stats-Core', :git => 'https://github.com/muxinc/stats-sdk-objc.git', :branch => 'nidhik/viewer-device-model'
```

Comment:
Note to self: remove this and use 'Mux-Stats-Core', '~>3.1.0' when that is released